README.md

```diff
@@ -6,8 +6,10 @@ This is a Rust crate that implements a [circular buffer], also known as cyclic
 buffer, circular queue or ring.
 
 This circular buffer has a fixed maximum capacity, does not automatically grow,
-and once its maximum capacity is reached, elements at the start of the buffer
-are overwritten. It's useful for implementing fast FIFO (_first in, first out_)
+and by default once its maximum capacity is reached, elements at the start of the buffer
+are overwritten. There are also `push_*_or_panic` methods that panic on maximum capacity and `try_push_*` that return the pushed elements as an error.
+
+Circular buffer is useful for implementing fast FIFO (_first in, first out_)
```

Comment:
I would avoid mentioning these methods here in the README, firstly because this README is supposed to give a very high-level overview of what the crate is about, and these methods are very specialized and (in my opinion) cover only a few cases; secondly because this README actually does not mention any method at all. It contains an example at the bottom, but besides that, methods are not covered in the README, so it's odd to me that `try_push_back` is mentioned, while `push_back` is not.

Reply:
Ok. I'd prefer to leave "by default" mention.